.github/workflows/review-app.yml

```diff
@@ -0,0 +1,105 @@
+# inspired from https://github.com/dokku/github-action/blob/master/example-workflows/review-app.yaml
+
+name: Deploy review app
+
+on: pull_request
+
+jobs:
+  handle_review_app:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.site }}
+      cancel-in-progress: true
+    strategy:
+      matrix:
+        site:
+          - ecospheres
+          - meteo-france
+          - logistique
+    permissions:
+      deployments: write
+    if: github.event_name == 'pull_request'
+    steps:
+      - name: Cloning repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Start deployment
+        if: always()
+        uses: chrnorm/deployment-action@v2
+        id: deployment
+        with:
+          token: ${{ github.token }}
+          environment: ${{ matrix.site }}-preview
+          initial-status: in_progress
+          transient-environment: true
+
+      - name: Create the review app
+        if: github.event.action == 'opened'
+        uses: dokku/github-action@master
+        with:
+          command: review-apps:create
+          git_remote_url: ${{ secrets.REVIEW_APP_SSH_URL }}
+          review_app_name: deploy-preview-${{ github.event.pull_request.number }}--${{ matrix.site }}
```

Comment:
`deploy-preview-${{ github.event.pull_request.number }}--${{ matrix.site }}` is used quite a bit -> extract into a variable?

Reply:
So as `env` it doesn't work? Too bad :(

Reply:
Nope, cause github.* and matrix.* are not available at the level where env are defined.